packaging/installer/kickstart.sh

```diff
@@ -1356,6 +1356,14 @@ check_special_native_deps() {
   fi
 }
 
+cleanup_apt_cache() {
+    cache_dir="/var/cache/apt/archives"
+
+    if [ -d "${cache_dir}" ]; then
+        find "${cache_dir}" -t file -name 'netdata*.deb' -delete
```

Comment:
Do we need `run_as_root` here?

Reply:
Quite so, I completely forgot about that.